docs/src/app/components/pages/customization/themes.md

```diff
@@ -0,0 +1,229 @@
+We changed how themes work in v0.12.0 (check out
```

Comment:
That will be far enough back by the time 0.15.0 is released that we can probably the reference to it changing.